### DISCUSSION TITLE
Does flux cli provide ways to modify helm-controller images repo or Scheduling to which node

### DISCUSSION THREAD

**Discussion by olivercp3** [fluxcd/flux2#3641] [Q&A] [answered]
thanks

**Comment by hiddeco** [comment, marked as answer]
See https://fluxcd.io/flux/cheatsheets/bootstrap/#how-to-customize-flux on how to make modifications to (any) Deployment as part of the Flux bootstrap process.

**Reply by olivercp3** [reply]
ty!
